actors/builtin/power/power_test.go

```diff
@@ -441,15 +441,16 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 
 		power64Unit, err := builtin.ConsensusMinerMinPower(abi.RegisteredPoStProof_StackedDrgWindow64GiBV1)
 		require.NoError(t, err)
-		assert.True(t, power64Unit.GreaterThan(powerUnit))
+
+		belowLimitUnit := big.Div(powerUnit, big.NewInt(2))
```

Comment:
Remove this subtest which is testing deprecated behavior

Reply:
Is it deprecated? I agree that the test name is wrong / the test can perhaps be simplified, but the sequence of "assert well below minPower isn't aded" -> "assert 1 byte below minPower isn't added" -> "assert precisely at minPower is added" seems valuable to me (and a quick skim didn't reveal the same sequence being tested somewhere else).

Reply:
I was only skimming the other tests and they seemed to be testing something similar.  I trust your judgement.  If this test is sufficiently different then rename the subtest, remove any parts related to different proof types and fix the comment on line 438.